01/app.js

```diff
@@ -7,8 +7,19 @@ userNameForm.addEventListener('submit', function(e) {
     const firstName = e.target.elements.firstName;
     const lastName = e.target.elements.lastName;
 
-    const newUser = document.createElement('li');
-    newUser.classList.add('user-list__person');
-    newUser.innerText = firstName.value + ' ' + lastName.value;
-    usersList.appendChild(newUser);
+    if(firstName.value === '' && lastName.value === '') {
+        alert('Wpisz imię i nazwisko!');
+    } else if(firstName.value === '') {
+        alert('Wpisz imię!');
+    } else if(lastName.value === '') {
+        alert('Wpisz nazwisko!');
```

Comment:
Możesz też utworzyć dodatkową zmianą o nazwie `errors`, która jest tablicą i sprawdzać niezależnie każde z pól.
Wtedy uzależnić dodawanie od tego czy ta tablica jest pusta. Nie będziesz musiał się przejmować tyloma warunkami - może teraz tego nie widać, ale przy większej ilości pól mógłby być to problem.